libs/api/src/lib/memberContext.ts

```diff
@@ -663,14 +667,16 @@ export class MemberContext implements MemberContextInterface {
       )
     }
 
+    const now = new Date()
+
     const subscription = await prisma.subscription.create({
       data: {
         userID,
-        startsAt: startsAt ? startsAt : new Date(),
+        startsAt,
         modifiedAt: new Date(),
         paymentMethodID: paymentMethodId,
         paymentPeriodicity,
-        paidUntil: null,
```

Comment:
Why can't that be null? For example if someone buys a subscription then the paidUntil is null at the beginning before it is paid or did you change something there?